src/CommandHandler/Wishlist/CopySelectedProductsToOtherWishlistHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\CommandHandler\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\CopySelectedProductsToOtherWishlist;
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\WishlistItemInterface;
+use BitBag\SyliusWishlistPlugin\Controller\Action\AddProductVariantToWishlistAction;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+final class CopySelectedProductsToOtherWishlistHandler
+{
+    private AddProductVariantToWishlistAction $addProductVariantToWishlistAction;
```

Comment:
It's not a good practice to use controllers (the user interface layer) inside a command handler (application layer).